src/reducer.js

```diff
@@ -1,9 +1,10 @@
 import { combineReducers } from 'redux';
-import { counterReducer, listOfStudentsReducer } from './features';
+import { counterReducer, listOfStudentsReducer, listOfCoursesReducer } from './features';
 
 export const rootReducer = combineReducers({
     features: combineReducers({
         counter: counterReducer,
         listOfStudents: listOfStudentsReducer,
+        listOfCourses: listOfCoursesReducer,
```

Comment:
why do we save search value in the redux store?

Reply:
I have changed it. We use useState and useEffect for searching courses by name.